SimplenoteUITests/History.swift

```diff
@@ -0,0 +1,45 @@
+import XCTest
+
+class History {
+
+    class func close() {
+        guard app.staticTexts[UID.Button.restoreNote].waitForExistence(timeout: minLoadTimeout) else { return }
+        app.buttons[UID.Button.dismissHistory].tap()
+    }
+
+    class func setSliderPosition(position: CGFloat) {
+        app.sliders.element.adjust(toNormalizedSliderPosition: position)
+    }
+
+    class func restoreNote() {
+        app.buttons[UID.Button.restoreNote].tap()
+    }
+}
+
+class HistoryAssert {
+    class func historyShown() {
+        let closeHistoryButton = app.buttons[UID.Button.dismissHistory]
+        let restoreNoteButton = app.buttons[UID.Button.restoreNote]
+
+        XCTAssertTrue(closeHistoryButton.waitForExistence(timeout: minLoadTimeout), UID.Button.dismissHistory + buttonNotFound)
+        XCTAssertTrue(restoreNoteButton.waitForExistence(timeout: minLoadTimeout), UID.Button.restoreNote + buttonNotFound)
+    }
+
+    class func historyDismissed() {
+        let closeHistoryButton = app.buttons[UID.Button.dismissHistory]
+
+        XCTAssertFalse(closeHistoryButton.exists, UID.Button.dismissHistory + buttonNotFound)
```

Comment:
`buttonNotFound` message is used here, while something like `buttonNotAbsent` would be correct.

Also, since we are checking the both `Dismiss` and `Restore` buttons presence in the `HistoryAssert.historyShown()` method, why not do the same here?

```suggestion
        let closeHistoryButton = app.buttons[UID.Button.dismissHistory]
        let restoreNoteButton = app.buttons[UID.Button.restoreNote]

        XCTAssertFalse(closeHistoryButton.exists, UID.Button.dismissHistory + buttonNotAbsent)
        XCTAssertFalse(restoreNoteButton.exists, UID.Button.restoreNote + buttonNotAbsent)
```